WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1435,6 +1453,12 @@ public void onImageSettingsRequested(EditorImageMetaData editorImageMetaData) {
         MediaSettingsActivity.showForResult(this, mSite, editorImageMetaData);
     }
 
+    @Override
+    public void onAudioSettingsRequested(String mediaModelId) {
+        MediaModel mediaModel = mMediaStore.getSiteMediaWithId(mSite, Integer.parseInt(mediaModelId));
```

Comment:
just tested uploading an audio file, after the upload finished I tapped on it and made the app crash with this log:
```
07-31 09:48:18.055 22747-22747/org.wordpress.android E/InputEventReceiver: Exception dispatching input event.
07-31 09:48:18.055 22747-22747/org.wordpress.android E/MessageQueue-JNI: Exception in MessageQueue callback: handleReceiveCallback
07-31 09:48:18.059 22747-22747/org.wordpress.android E/MessageQueue-JNI: java.lang.NumberFormatException: s == null
        at java.lang.Integer.parseInt(Integer.java:570)
        at java.lang.Integer.parseInt(Integer.java:643)
        at org.wordpress.android.ui.posts.EditPostActivity.onAudioSettingsRequested(EditPostActivity.java:1458)
        at org.wordpress.android.editor.AztecEditorFragment.onMediaTapped(AztecEditorFragment.java:1722)
        at org.wordpress.android.editor.AztecEditorFragment.onAudioTapped(AztecEditorFragment.java:1566)
        at org.wordpress.aztec.spans.AztecAudioSpan.onClick(AztecAudioSpan.kt:23)
        at org.wordpress.aztec.spans.AztecMediaClickableSpan.onClick(AztecMediaClickableSpan.kt:8)
        at org.wordpress.aztec.EnhancedMovementMethod.onTouchEvent(EnhancedMovementMethod.kt:77)

```

It is possible that the `mediaModelId` is null here, or something else (not an Int).

Anyway, considering this a bit more made me realize that we don't really want to trigger the audio settings here as we'd do for Images, but rather treat tapping on a playable item the same way we do for video. I'll make a separate comment on the appropriate line of code.